README.md

```diff
@@ -65,21 +65,43 @@ Cloning and Startup
 
   ``git clone https://github.com/openedx/frontend-app-gradebook.git``
 
-2. Install npm dependencies:
+2. Use node v20.x. The current version of the micro-frontend build scripts
+   supports node 20. Using other major versions of node may work, but this is
+   unsupported. For convenience, this repository includes an .nvmrc file to help
+   in setting the correct node version via [nvm](https://github.com/nvm-sh/nvm).
+
+3. Stop the Tutor devstack, if it's running:
+
+  ``tutor dev stop``
+
+4. Next, we need to tell Tutor that we're going to be running this repo in development mode, and it should be excluded from the mfe container that otherwise runs every MFE. Run this:
+
+  ``tutor mounts add /path/to/frontend-app-gradebook``
+
+5. Start Tutor in development mode. This command will start the LMS and Studio,
+   and other required MFEs like ``authn`` and ``account``, but will not start the
+   Gradebook MFE, which we're going to run on the host instead of in a container
+   managed by Tutor. Run:
+
+   ``tutor dev start lms cms mfe``
+
+## Startup
+
+1. Install npm dependencies:
 
   ``cd frontend-app-gradebook && npm install``
 
-3. Start the dev server:
+2. Start the dev server:
```

Comment:
`npm start` will not work here. This MFE does not yet have an `npm run dev` script in `package.json`. I think landing a PR adding a `dev` script to the `package.json` (should be easy to code and test) before landing these docs changes is likely a decent idea.

Reply:
I'm probably not the right person to do that 😅 but would you like me to ticket that up?

Reply:
https://github.com/openedx/frontend-app-gradebook/pull/438